drivers/bluetooth/hci/spi.c

```diff
@@ -460,6 +501,9 @@ static int bt_spi_open(void)
 static const struct bt_hci_driver drv = {
 	.name		= "BT SPI",
 	.bus		= BT_HCI_DRIVER_BUS_SPI,
+#if defined(CONFIG_BT_BLUENRG_ACI)
+	.quirks		= BT_QUIRK_NO_RESET,
```

Comment:
When this quirk is set, the assumption is that the HCI driver takes care of HCI_Reset before open() returns. I don't see that in your patch anywhere? Also, you could add a bit more descriptive commit message (explaining why this all is needed).

Reply:
I update to commit message to explain it. At least with my test I have seen that when BlueNRG are initialised, it not necessary to make a 'HCI_reset', it's why I just force the hardware on controller mode after it's initialised.

Reply:
Yep, looks good now